src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './adapter';
+export * from './casbinMongoRule';
+export * from './casbinRule';
```

Comment:
There must be `export default` here for compatible with previous version.

It looks like this,
```ts
export { TypeORMAdapterOptions } from './adapter';
export * from './casbinMongoRule';
export * from './casbinRule';

import TypeORMAdapter from './adapter';

export default TypeORMAdapter;
```